atomic_reactor/cli/job.py

```diff
@@ -5,7 +5,38 @@
 This software may be modified and distributed under the terms
 of the BSD license. See the LICENSE file for details.
 """
+import logging
+
+from atomic_reactor.config import Configuration, get_openshift_session
+from atomic_reactor.utils import remote_host
+
+
+logger = logging.getLogger(__name__)
 
 
 def remote_hosts_unlocking_recovery(job_args):
-    return
+    config = Configuration(config_path=job_args['config_file'])
+    osbs = get_openshift_session(config, job_args['namespace'])
+
+    remote_host_pools = config.remote_hosts.get("pools")
+
+    for platform in remote_host_pools.keys():
```

Comment:
should remote_host_pools be `remote_host_pools = config.remote_hosts.get("pools", {})` to avoid a potential NoneType error? (darn, I was hoping mypy would catch this)

Reply:
pools is required in schema

Reply:
well I'd rather let it blow, this is job for unlocking zombie slots on remote hosts

if you don't have any remote hosts defined it should tbh fail

Reply:
This makes me think - what about the source container namespace? Will the job be deployed in there as well? Probably shouldn't right?

Reply:
I've made changes in playbooks, so it is deployed just for binary namespace